Branding/Ghostery/de.lproj/ForgetMode.strings

```diff
@@ -0,0 +1,25 @@
+/* 
+  ForgetMode.strings
+  UserAgent
+
+  Created by Shahen Antonyan on 5/26/20.
+  Copyright © 2020 Cliqz. All rights reserved.
+*/
+
+/* Action label on share extension to immediately open page in a private tab. */
+"ForgetMode.ShareExtension.OpenInPrivateTab" = "In neuem Vergessen-Modus Tab öffnen";
```

Comment:
```suggestion
"ForgetMode.ShareExtension.OpenInPrivateTab" = "In neuem Ghost-Tab öffnen";
```

Reply:
Should it say "neuem" ("new")? If so, it's the same string as for `ForgetMode.ContextMenu.OpenInNewPrivateTab`.